iquip/apps/dataviewer.py

```diff
@@ -1157,7 +1163,7 @@ def updateMainPlot(self, axis: Sequence[int], dataType: DataPointWidget.DataType
             return
         reduce = self._reduceFunction(dataType)
         data, axes = self.policy.extract(axis, reduce)
-        self.frame.mainPlotWidget.setData(data, axes)
+        self.frame.mainPlotWidget.setData(data, axes, dataType)
```

Comment:
I think we should `setYRange()` here, i.e., `self.frame.mainPlotWidget.viewer().plotItem.setYRange(0, 1)`,
since the `MainPlotWidget` is independent to the `DataType`.
What do you think of this?

Reply:
I see! I updated it as you suggested.